Documentation for Radius + Flux integration (#1408)
radius-project/docs: docs/content/guides/deploy-apps/gitops/overview/_index.md

Reviewer: rename this to `.../overview/index.md` and also add `docs/content/guides/deploy-apps/gitops/_index.md`(similar to https://github.com/radius-project/docs/blob/v0.44/docs/content/guides/author-apps/dapr/_index.md)

---
type: docs
title: "Overview: Radius and GitOps"
linkTitle: "Overview"
description: "Learn about how to leverage GitOps tools to deploy Radius resources"
weight: 100
categories: "Overview"
tags: ["gitops", "continuous", "delivery", "deployment"]
---

> Radius integrates seamlessly with GitOps to enhance the continuous deployment and management of cloud-native applications and infrastructure. Currently, Radius supports [Flux](https://fluxcd.io/), with [ArgoCD](https://argoproj.github.io/cd/) support coming in the future.

Reviewer: Adding an edit to what I originally wrote :)
```suggestion
Radius integrates seamlessly with GitOps to enhance the continuous deployment and management of cloud-native applications and infrastructure. Currently, Radius has buillt-in support for [Flux](https://fluxcd.io/) and can be extended to support other platforms.
```

3. **Drift Detection and Reconciliation**: The GitOps tool continuously monitors the production environment for any drift from the desired state defined in the Git repository. If any discrepancies are detected, Radius will be triggered to automatically reconcile the state to match the repository, maintaining consistency and reliability.

4. **Enhanced Security and Compliance**: By using Git as the single source of truth, users can ensure that all changes are tracked and auditable, even if they are executed by Radius. This enhances security and compliance, as every change is documented and can be reviewed.

5. **Scalability and Flexibility**: Radius's integration with GitOps tools supports scalable and flexible deployment strategies. Whether deploying to a single cluster or multiple clusters across different environments, Radius provides a consistent and reliable deployment process.

## Getting Started

Today, Radius has integrated first-class support for [Flux](https://fluxcd.io/), a popular GitOps tool. Flux is designed to work with Kubernetes and provides a powerful set of features for managing applications and infrastructure through Git.

> To get started with using Radius and Flux, check out the [Radius + Flux Tutorial]({{< ref "tutorials/gitops-flux" >}}) or the [how-to guide for Radius + Flux]({{< ref "guides/deploy-apps/gitops/flux" >}}).

Reviewer: Need to link to the actual page for Flux How-To

> ## Future Work
> 
> In the future, Radius will also support [ArgoCD](https://argoproj.github.io/cd/). If this is of interest to you, please comment on the [issue](https://github.com/radius-project/radius/issues/8078) to help us prioritize this work.

Reviewer: remove this as docs is not the place for talking about roadmap stuff

```suggestion
```
